resources/js/Directives/getinvolvedDirective.js

```diff
@@ -31,6 +31,12 @@ app.directive('getinvolved', ['$http', function ($http) {
             $scope.loadMore = function () {
                 $scope.totalDisplayed += 20;
             };
+
+            $scope.nomoreItems = function () {
+                if(self.contributors){
```

Comment:
Add space before the first paren and after the second paren so it follows the style of the other files.